registration/pages/_app.tsx

```diff
@@ -0,0 +1,11 @@
+import { ApolloProvider } from '@apollo/react-hooks'
+import { apolloClient } from '../lib/apollo'
+
+
+// documentation for _app is almost non-existent, but i found this 
+// https://github.com/zeit/next.js/blob/canary/errors/app-container-deprecated.md
+export default ({ Component, pageProps }) => {
```

Comment:
This works without inheriting `App`? :scream:

Reply:
yes, tested

Reply:
should inherit, ur rite